staging/src/k8s.io/apiserver/pkg/storage/util.go

```diff
@@ -29,6 +29,13 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 )
 
+const (
```

Comment:
Let's move this one to a separate cleanup PR (together with the new func and test).

Reply:
https://github.com/kubernetes/kubernetes/pull/120872

Reply:
I will rebase this PR once 120872 merges unless git is able to figure it out automatically.